ADVANCED.md

```diff
@@ -811,16 +811,26 @@ return (
-| `autocompleteQuery` | Object                                                                  | optional  |         | Configuration options for the main search query.                                                                                                                                                                                                         |
-|                     | - `results` - [Query Config](#query-config)                             |           |         | Configuration options for results query, used by autocomplete.                                                                                                                                                                                           |
-|                     | - `suggestions` - [Suggestions Query Config](#suggestions-query-config) |           |         | Configuration options for suggestions query, used by autocomplete.                                                                                                                                                                                       |
-| `debug`             | Boolean                                                                 | optional  | false   | Trace log actions and state changes.                                                                                                                                                                                                                     |
-| `initialState`      | Object                                                                  | optional  |         | Set initial [State](#headless-core#state) of the search. Any [Request State](#headless-core#1-request-state) can be set here. This is useful for defaulting a search term, sort, etc.<br/><br/>Example:<br/>`{ searchTerm: "test", resultsPerPage: 40 }` |
-| `searchQuery`       | [Query Config](#query-config)                                           | optional  |         | Configuration options for the main search query.                                                                                                                                                                                                         |
-| `trackUrlState`     | boolean                                                                 | optional  |         | By default, [Request State](#headless-core#1-request-state) will be synced with the browser url. To turn this off, pass `false`.                                                                                                                         |
+apiConnector: connector,
+// Example of overriding a connector handler
+onResultClick: () => console.log("hi"),
+// Example of handler as middleware
+onSearch: (state, queryConfig, next) => next(state, queryConfig)
```

Comment:
What are these examples? They don't seem to belong here?

Reply:
🤦‍♂ Good catch, those are NOT supposed to be there. Will update.